mks/model/gba.py

```diff
@@ -0,0 +1,36 @@
+import csv
+import os.path
+
+from mks.service.config import STATIC_DIR
+
+GBA_STATIC_DIR = os.path.join(STATIC_DIR, 'gba')
+
+lookup_landen = {}
+lookup_gemeenten = {}
+lookup_geslacht = {
+    'M': "Man",
```

Comment:
Is this a convention? Using different quote characters for key => value ?

Reply:
Nope not a convention.
I generally do not care too much about quotation. I'll change it to be the same.